sys-kernel/asahi-sources/asahi-sources-6.11.6_p2.ebuild

```diff
@@ -42,17 +45,10 @@ KV_FULL="${PVR/_p/-asahi-}"
 S="${WORKDIR}/linux-${KV_FULL}"
 
 KEYWORDS="arm64"
-IUSE="rust"
```

Comment:
rust is no longer conditional -- is that intentional? I assume it is and it's fine, just noting it.

Reply:
As noted in the commit message, we have a number of drivers and abstractions in Rust now that are essentially required for a functioning system (no one is using these things without the GPU, SEP is going to be rust, AOP is going to be rust, DCP is probably going to get a rewrite soon, etc.).